app/assets/javascripts/discourse/templates/preferences/account.hbs

```diff
@@ -71,9 +71,7 @@
     {{#if model.second_factor_enabled}}
       {{i18n 'user.second_factor.disable'}}
     {{else}}
-      {{i18n 'enable'}}
-      <a href {{action "showTwoFactorModal"}}>{{showTwoFactorModalText}}</a>
-      {{i18n 'user.second_factor.enable'}}
+      <span onclick={{action 'showTwoFactorModal'}}>{{{i18n 'user.second_factor.enable'}}}</span>
```

Comment:
something does not feel right to me using `span onclick` why doing we need hyperlink inception here?

Reply:
Because I wanted to replace 3 concatenated strings (`enable`, `user.second_factor.title`, `user.second_factor.enable`) with just one. Otherwise it's really hard to translate. The 2nd one is currently even converted into lower case.

Unfortunately the string should contain a link that calls an action and that's probably why @tgxworld used 3 strings instead of just 1.

![image](https://user-images.githubusercontent.com/473736/40601852-f86a63d2-6256-11e8-9212-af3ed0f883f5.png)

I know my solution is kind of a hack. If there is a better solution I'm all ears! Unfortunately `{{{i18n 'user.second_factor.enable' link=(action "showTwoFactorModal")}}}` didn't work.